model-optimizer/unit_tests/extensions/ops/split_test.py

```diff
@@ -188,3 +188,76 @@ def test_splitv_2_zero_not_last(self):
         self.assertTrue(node.out_port(4).disconnected())
         self.assertTrue(node.out_port(3).disconnected())
         self.assertTrue(np.all(node.split_lengths == np.array([2, 13, 10])))
+
```

Comment:
Add one more empty line.

In general, apply pep8 to the whole file.

Reply:
Done.